src/vm/gcholder.h

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef _GCHOLDER_H_
+#define _GCHOLDER_H_
+
+#include "gcinterface.h"
+
+// The singular heap instance.
+GPTR_DECL(IGCHeap, g_pGCHeap);
+
+// GCHeapHolder provides a number of static methods
+// that operate on the global heap instance. It can't be
+// instantiated.
+class GCHeapHolder {
```

Comment:
>GCHeapHolder [](start = 6, length = 12)

normally I wouldn't make requests like this because it's fairly cosmetic but Holder is already used extensively in the VM code to mean something with an automatic destructor. would be better to call it GCHeapUtilities or something that doesn't have the word Holder....